source/jormungandr/tests/overlapping_routing_tests.py

```diff
@@ -34,6 +34,22 @@
 from .check_utils import *
 
 
+@dataset({"main_routing_test": {'is_free': False}, "empty_routing_test": {'is_free': True}})
+class TestOverlappingBestCoverage(AbstractTestFixture):
+    """
+    Test the answer if 2 coverages are overlapping by choosing the best one according to the instances comparator
```

Comment:
Minor-minor: add a comment on what is the other possible coverage and its output can be nice.

Reply:
That's minor-minor-minor-minor : looking at the PR, the instances comparator is explained several times